### PR TITLE
chore: Update About The App section and rename it to 'About Us'

### LINKED ISSUE #1903: Update About The App section and rename it to 'About Us'
**Describe the chore**
- Update About The App section to add more details like version details and other links.
- Rename it to `About Us`.

**Would you like to work on the issue?**
- [x] Yes
- [ ] No

### PR DESCRIPTION
**Pull request by ShridharGoel**
Fixes #1903 

GIF for the change:

About Us

### CODE REVIEW

**Comment by iamareebjamal** [comment]
1. What's the increase in app size
2. Everything is hardcoded. It should be configurable

File: app/src/main/java/com/eventyay/organizer/core/about/AboutUsFragment.java [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+        shareElement.setOnClickListener(v -> {
+            Intent sharingIntent = new Intent();
+            sharingIntent.setAction(Intent.ACTION_SEND);
+            sharingIntent.setType("text/plain");
+            String shareSub = getString(R.string.msg_check_out);
+            String shareBody = getString(R.string.msg_check_this_out) + getString(R.string.play_store_link);
+            String shareTitle = getString(R.string.msg_share_using);
+            sharingIntent.putExtra(Intent.EXTRA_SUBJECT, shareSub);
+            sharingIntent.putExtra(Intent.EXTRA_TEXT, shareBody);
+            startActivity(Intent.createChooser(sharingIntent, shareTitle));
```
Comment (iamareebjamal) [inline app/src/main/java/com/eventyay/organizer/core/about/AboutUsFragment.java]: Duplicate sharing code

Comment (ShridharGoel) [reply app/src/main/java/com/eventyay/organizer/core/about/AboutUsFragment.java]: How?

Comment (ShridharGoel) [reply app/src/main/java/com/eventyay/organizer/core/about/AboutUsFragment.java]: I've removed it from the Device Settings.

Comment (ShridharGoel) [reply app/src/main/java/com/eventyay/organizer/core/about/AboutUsFragment.java]: And the sharing code in Utils is for sharing an event.

Comment (iamareebjamal) [reply app/src/main/java/com/eventyay/organizer/core/about/AboutUsFragment.java]: OK

Comment (ShridharGoel) [reply app/src/main/java/com/eventyay/organizer/core/about/AboutUsFragment.java]: > Everything is hardcoded. It should be configurable

Configurable from where?

**Comment by ShridharGoel** [comment]
> Everything is hardcoded. It should be configurable

Configurable from where?

**Comment by iamareebjamal** [comment]
Package name can be queried programmatically. App name as well. Whatever cannot be coded should be configurable from build.gradle

**Comment by ShridharGoel** [comment]
Okay

File: app/src/main/java/com/eventyay/organizer/core/about/AboutUsFragment.java [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+            acknowledgementDecider.openAcknowledgementsSection(getContext());
+        });
+
+        AboutPage aboutPage = new AboutPage(getContext())
+            .isRTL(false)
+            .setImage(R.mipmap.ic_launcher)
+            .setDescription(getString(R.string.about_us_description))
+            .addItem(new Element("Version " + BuildConfig.VERSION_NAME, R.drawable.ic_info))
+            .addGroup("Connect with us")
+            .addGitHub("fossasia/open-event-organizer-android")
```
Comment (iamareebjamal) [inline app/src/main/java/com/eventyay/organizer/core/about/AboutUsFragment.java]: Use git plugin in gradle for repo name

File: app/src/main/java/com/eventyay/organizer/core/about/AboutUsFragment.java [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+            .addPlayStore("com.eventyay.organizer")
+            .addWebsite(getString(R.string.FRONTEND_HOST))
+            .addFacebook("eventyay")
+            .addTwitter("eventyay")
```
Comment (iamareebjamal) [inline app/src/main/java/com/eventyay/organizer/core/about/AboutUsFragment.java]: Make it configurable

File: app/src/main/java/com/eventyay/organizer/core/about/AboutUsFragment.java [hunk 4]
```diff
@@ -0,0 +1,112 @@
...
+            .addYoutube("UCQprMsG-raCIMlBudm20iLQ")
```
Comment (iamareebjamal) [inline app/src/main/java/com/eventyay/organizer/core/about/AboutUsFragment.java]: Same

**Comment by ShridharGoel** [comment]
> What's the increase in app size

No increase.

File: app/src/main/java/com/eventyay/organizer/core/about/AboutUsFragment.java (hunk 2 above)
Comment (ShridharGoel) [reply app/src/main/java/com/eventyay/organizer/core/about/AboutUsFragment.java]: Can you please explain about this? I've done the other changes.

Comment (iamareebjamal) [reply app/src/main/java/com/eventyay/organizer/core/about/AboutUsFragment.java]: Ignore for now

File: app/src/main/java/com/eventyay/organizer/core/about/AboutUsFragment.java (hunk 4 above)
Comment (ShridharGoel) [reply app/src/main/java/com/eventyay/organizer/core/about/AboutUsFragment.java]: Done

File: app/src/main/java/com/eventyay/organizer/core/about/AboutUsFragment.java (hunk 3 above)
Comment (ShridharGoel) [reply app/src/main/java/com/eventyay/organizer/core/about/AboutUsFragment.java]: Done

File: app/build.gradle [hunk 5]
```diff
@@ -75,6 +78,9 @@ android {
             buildConfigField "String", "DEFAULT_BASE_URL", '"https://open-event-api-dev.herokuapp.com/v1/"'
             resValue "string", "default_base_url", '"https://open-event-api-dev.herokuapp.com/v1/"'
             resValue "string", "FRONTEND_HOST", "open-event-fe.netlify.com"
+            resValue "string", "FACEBOOK_ID", "eventyay"
+            resValue "string", "TWITTER_ID", "eventyay"
+            resValue "string", "YOUTUBE_ID", "UCQprMsG-raCIMlBudm20iLQ"
```
Comment (iamareebjamal) [inline app/build.gradle]: Duplication

Comment (ShridharGoel) [reply app/build.gradle]: Updated.

**Pull request merged**